pinot-druid/src/test/java/org/apache/pinot/druid/data/readers/TestDruidSegmentRecordReader.java

```diff
@@ -0,0 +1,71 @@
+package org.apache.pinot.druid.data.readers;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.pinot.common.data.Schema;
+import org.apache.pinot.core.data.GenericRow;
+import org.apache.pinot.core.indexsegment.generator.SegmentGeneratorConfig;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testng.annotations.Test;
+
+
+public class TestDruidSegmentRecordReader {
```

Comment:
Let's improve testing code in general :) 

1. add more tests as you listed in `TODO`
2. make them complete (e.g. use `Assert.xxx` instead of print out).